src/test/java/org/broadinstitute/hellbender/utils/recalibration/covariates/ContextCovariateUnitTest.java

```diff
@@ -34,7 +35,8 @@ public void testSimpleContexts() {
         final SAMFileHeader header = ArtificialReadUtils.createArtificialSamHeader();
 
         for(int i = 0; i < 10; i++) {
-            final GATKRead read = ArtificialReadUtils.createRandomRead(header, 1000);
+            final GATKRead read = ArtificialReadUtils.createRandomRead(header, 1000,false);
```

Comment:
All tests that generate random data using the rng returned by `Utils.getRandomGenerator()` should call `Utils.resetRandomGenerator()` at the top to make them deterministic/repeatable (I know this is not your problem, but since you're in here...)